status/format.go

```diff
@@ -170,19 +179,23 @@ func FormatDetails(taskRun *helpers.TaskRun) (string, error) {
 	}
 
 	if result == nil {
-		return "", nil
+		var emoji string
+		taskSucceededReason := taskRun.GetStatusCondition(string(apis.ConditionSucceeded)).GetReason()
+		switch taskSucceededReason {
+		case "Succeeded":
+			emoji = ":heavy_check_mark:"
+		case "Failed":
+			emoji = ":x:"
+		default:
+			emoji = ":question:"
+		}
+		return fmt.Sprintf(emoji+" Reason: %s", taskSucceededReason), nil
 	}
 
 	if result.ValidationError != nil {
 		return fmt.Sprintf("Invalid result: %s", result.ValidationError), nil
 	}
-
-	if result.TestOutput == nil {
```

Comment:
I am just curious about this removal. Will this condition not occurs?

Reply:
So I need to add clarification here, this condition is true also when `result.ValidationError != nil`, meaning it will never reach it, since its covered by previous condition.